pyproject.toml

```diff
@@ -26,7 +26,7 @@ classifiers = [
 ]
 requires-python = ">= 3.11, < 4"
 dependencies = [
-  "frequenz-api-common >= 0.4.0, < 0.5",
+  "frequenz-api-common@git+https://github.com/frequenz-floss/frequenz-api-common",
```

Comment:
I guess this can be put back to point to a proper release  now that 0.5.0 is released.

Reply:
Oops, had forgotten about this. Thanks for pointing out. Fixed.